app/src/main/java/lt/vilnius/tvarkau/analytics/RemoteAnalytics.kt

```diff
@@ -14,9 +14,18 @@ class RemoteAnalytics(appContext: Context) : Analytics {
     }
 
     private val PARAM_PROBLEM_STATUS = "problem_status"
+    private val PARAM_FRAGMENT_ACTIVITY = "activity"
 
     override fun trackCurrentFragment(activity: Activity, fragment: Fragment) {
-        analytics.setCurrentScreen(activity, fragment.javaClass.simpleName, null)
+        val fragmentName = fragment.javaClass.simpleName
+
+        Bundle().run {
```

Comment:
This `run {}` really doesn't help to readability or give any other benefit.
```
val bundle = Bundle()
bundle.putString(PARAM_FRAGMENT_ACTIVITY, activity.javaClass.simpleName)
analytics.logEvent("open_$fragmentName", bundle)
```
looks equally good.